tgui/packages/tgui/interfaces/DropshipWeaponsConsole.tsx

```diff
@@ -0,0 +1,358 @@
+            <Stack.Item>
+              <svg height="200" width="200">
+                <DrawShipOutline />
+                {weapons.map((x) => (
+                  <DrawWeapon key={x.mount_point} weapon={x} />
+                ))}
+              </svg>
+            </Stack.Item>
+            <Stack.Item>
+              <Stack>
+                {support.map((x) => (
+                  <Stack.Item key={x.name}>{x.name}</Stack.Item>
+                ))}
+              </Stack>
+            </Stack.Item>
+          </Stack>
+        </Stack.Item>
+        <Stack.Item>
+          <Flex
+            className="WeaponPanelRight"
+            direction="column"
+            justify="space-around">
+            <Flex.Item>
+              <WeaponStatsPanel
+                slot={3}
+                weapon={weapons.find((x) => x.mount_point === 3)}
+              />
+            </Flex.Item>
+            <Flex.Item>
+              <Divider />
+            </Flex.Item>
+            <Flex.Item>
+              <WeaponStatsPanel
+                slot={4}
+                weapon={weapons.find((x) => x.mount_point === 4)}
+              />
+            </Flex.Item>
+          </Flex>
+        </Stack.Item>
+      </Stack>
+    </Box>
+  );
+};
+
+const LcdPanel = (props, context) => {
+  const { data } = useBackend<DropshipProps>(context);
+  return (
+    <Box className="NavigationMenu">
+      <DropshipWeaponsPanel equipment={data.equipment_data} />
+    </Box>
+  );
+};
+
+const FiremissionSimulationPanel = (props, context) => {
+  return (
+    <Box className="NavigationMenu">
+      <CasSim />
+    </Box>
+  );
+};
+
+const FiremissionsSimMfdPanel = (props: MfdProps, context) => {
+  const [panelState, setPanelState] = usePanelState(
+    props.panelStateId,
+    context
+  );
+  return (
+    <MfdPanel
+      panelStateId={props.panelStateId}
+      bottomButtons={[
+        {
+          children: 'BACK',
+          onClick: () => setPanelState(''),
+        },
+      ]}>
+      <FiremissionSimulationPanel />
+    </MfdPanel>
+  );
+};
+
+const WeaponsMfdPanel = (props, context) => {
+  return (
+    <MfdPanel panelStateId={props.panelStateId}>
+      <LcdPanel />
+    </MfdPanel>
+  );
+};
+
+const BaseMfdPanel = (props: MfdProps, context) => {
+  const [panelState, setPanelState] = usePanelState(
+    props.panelStateId,
+    context
+  );
+
+  return (
+    <MfdPanel
+      panelStateId={props.panelStateId}
+      topButtons={[
+        { children: 'EQUIP', onClick: () => setPanelState('equipment') },
+        {
+          children: 'F-MISS',
+          onClick: () => setPanelState('firemission'),
+        },
+        {},
+        {
+          children: 'TARGETS',
+          onClick: () => setPanelState('target-aquisition'),
+        },
+      ]}
+      bottomButtons={[
+        {},
+        { children: 'MAPS', onClick: () => setPanelState('map') },
+        { children: 'CAMS', onClick: () => setPanelState('camera') },
```

Comment:
These two buttons ought to be made mutually exclusive (as in if one panel is using MAPS, then the other panel loses that option) because the byondui at the moment does not like appearing in both places. You can probably ensure setPanelState just forces the other to exit if its in the same mode.

Otherwise you get this (right panel opened second so map swooped over there): 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/2a6a9c11-c4b2-47b3-b1ad-37768bac9f16)

Reply:
This is actually a real pain to do. The lazy approach would just be have two minimaps, one for each side. I'd like to migrate the tacmap over at some point.